api/urls.py

```diff
@@ -11,4 +11,6 @@
     url(r'^proposicoes/(?P<casa>[a-z]+)/(?P<id_ext>[0-9]+)/?$',
         views.ProposicaoDetail.as_view()),
     url(r'^proposicoes/?$', views.ProposicaoList.as_view()),
+    url(r'^energia/(?P<casa>[a-z]+)/(?P<id_ext>[0-9]+)/(?P<periodo>[0-9]+)/?$',
```

Comment:
Não seria melhor que a URI o formato:  ```/energia/senado/91341?periodo=90```?
https://stackoverflow.com/questions/4024271/rest-api-best-practices-where-to-put-parameters

Reply:
Eu não gosto de querystring porque acho mais feio e difícil de usar, mas acho que o Daniel tem razão nesse caso. Parece fazer mais sentido usar querystring aí mesmo...